src/mountebank.js

```diff
@@ -39,6 +42,7 @@ async function create (options) {
         validateImposterExists = middleware.createImposterValidator(imposters),
         prometheus = require('prom-client');
 
+    prometheus.register.clear();
```

Comment:
It seems the place of this code in the tests, but i don't know how `mbTest` can access directly `prom-client` from `mountebank` node_modules

Reply:
just `require('../../node_modules/prom-client')` and it's work, but maybe will not working inside CI?